app/src/server/api/internal/v1Api.router.ts

```diff
@@ -25,25 +25,17 @@ export const v1ApiRouter = createOpenApiRouter({
       z.object({
         trainingDataUrl: z.string(),
         huggingFaceModelId: z.string(),
-        baseModel: z.string(),
-        projectName: z.string(),
-        modelSlug: z.string(),
+        trainingConfig: z.record(z.unknown()),
```

Comment:
Why isn't this known?

Reply:
I hadn't generated the full axolotl config as a zod schema, but I decided it was worth the hassle, so this isn't `unknown` anymore.